Alignment/OfflineValidation/python/TkAlAllInOneTool/helperFunctions.py

```diff
@@ -159,3 +159,34 @@ def newfunction(*args, **kwargs):
             return newfunction(*args, **kwargs)
     newfunction.__name__ = function.__name__
     return newfunction
+
+def boolfromstring(string, name):
+    """
+    Takes a string from the configuration file
+    and makes it into a bool
```

Comment:
the config parser contains a function `getboolean` which might do what you want, but I don't mind to leave it like this for now :)